src/igvfd/schemas/changelogs/image.md

```diff
@@ -1,5 +1,9 @@
 ## Changelog for *`image.json`*
 
+### Minor changes since schema version 4
+
+* Add calculated `summary`.
```

Comment:
I think `summary` was previously already in this object (just defaulted to UUID). In other summary tickets I wrote: Update calculation of summary.

Reply:
fixed.